root/layout/components/BottomMenu.js

```diff
@@ -56,273 +47,267 @@ const LanguageMenu = ({
-      {'\xA0\u25BE'}
-    </span>
-    <ul>
-      <li>
-        <a href="/doc/Beginners_Guide">{l('Beginners Guide')}</a>
-      </li>
-      <li>
-        <a href="/doc/Style">{l('Style Guidelines')}</a>
-      </li>
-      <li>
-        <a href="/doc/How_To">{l('How Tos')}</a>
-      </li>
-      <li>
-        <a href="/doc/Frequently_Asked_Questions">{l('FAQs')}</a>
-      </li>
-      <li>
-        <a href="/doc/MusicBrainz_Documentation">
-          {l('Documentation Index')}
-        </a>
-      </li>
-      <li className="separator">
-        <a href="/doc/Edit_Types">{l('Edit Types')}</a>
-      </li>
-      <li>
-        <a href="/relationships">{l('Relationship Types')}</a>
-      </li>
-      <li>
-        <a href="/instruments">{l('Instrument List')}</a>
-      </li>
-      <li>
-        <a href="/genres">{l('Genre List')}</a>
-      </li>
-      <li className="separator">
-        <a href="/doc/Development">{l('Development')}</a>
-      </li>
-    </ul>
-  </li>
+  <MenuDropdown
+    className="documentation"
+    id="documentation-menu-dropdown"
+    label={l('Documentation')}
+  >
+    <a className="dropdown-item" href="/doc/Beginners_Guide">
+      {l('Beginners Guide')}
+    </a>
+    <a className="dropdown-item" href="/doc/Style">
+      {l('Style Guidelines')}
+    </a>
+    <a className="dropdown-item" href="/doc/How_To">
+      {l('How Tos')}
+    </a>
+    <a className="dropdown-item" href="/doc/Frequently_Asked_Questions">
+      {l('FAQs')}
+    </a>
+    <a className="dropdown-item" href="/doc/MusicBrainz_Documentation">
+      {l('Documentation Index')}
+    </a>
+    <div className="dropdown-divider" />
+    <a className="dropdown-item" href="/doc/Edit_Types">
+      {l('Edit Types')}
+    </a>
+    <a className="dropdown-item" href="/relationships">
+      {l('Relationship Types')}
+    </a>
+    <a className="dropdown-item" href="/instruments">
+      {l('Instrument List')}
+    </a>
+    <a className="dropdown-item" href="/genres">
+      {l('Genre List')}
+    </a>
+    <div className="dropdown-divider" />
+    <a className="dropdown-item" href="/doc/Development">
+      {l('Development')}
+    </a>
+  </MenuDropdown>
 );
 
-const BottomMenu = ({$c}: {+$c: CatalystContextT}) => {
-  const serverLanguages = $c.stash.server_languages;
-  return (
-    <div className="bottom">
-      <ul className="menu">
-        <AboutMenu />
-        <ProductsMenu />
-        <SearchMenu />
-        {$c.user_exists ? <EditingMenu /> : null}
-        <DocumentationMenu />
-        {serverLanguages && serverLanguages.length > 1 ? (
-          <LanguageMenu
-            currentBCP47Language={$c.stash.current_language.replace('_', '-')}
-            serverLanguages={serverLanguages}
-          />
-        ) : null}
-      </ul>
-    </div>
-  );
+type Props = {
+  +$c: CatalystContextT,
+  +currentLanguage: string,
+  +serverLanguages: ?$ReadOnlyArray<ServerLanguageT>,
 };
 
-export default withCatalystContext(BottomMenu);
```

Comment:
Why did you decide to pass $c as a prop rather than use withCatalystContext?

Reply:
From what I can see, because the `Header` already uses `withCatalystContext`, so `BottomMenu` isn't referenced from anywhere we can't easily pass `$c` through already.